app/Console/Kernel.php

```diff
@@ -13,25 +32,25 @@ class Kernel extends ConsoleKernel
      * @var array
      */
     protected $commands = [
-        'App\Console\Commands\CalculateStatistics',
-        'App\Console\Commands\Deactivate2FA',
-        'App\Console\Commands\ExportAll',
-        'App\Console\Commands\GetVersion',
-        'App\Console\Commands\ImportCSV',
-        'App\Console\Commands\ImportVCards',
-        'App\Console\Commands\LangGenerate',
-        'App\Console\Commands\PingVersionServer',
-        'App\Console\Commands\SendReminders',
-        'App\Console\Commands\SendStayInTouch',
-        'App\Console\Commands\SentryRelease',
-        'App\Console\Commands\SetupProduction',
-        'App\Console\Commands\SetupTest',
-        'App\Console\Commands\SetupFrontEndTest',
-        'App\Console\Commands\SetPremiumAccount',
-        'App\Console\Commands\SetUserAdmin',
-        'App\Console\Commands\Update',
-        'App\Console\Commands\MigrateDatabaseCollation',
-        'App\Console\Commands\OneTime\MoveAvatars',
+        CalculateStatistics::class,
+        Deactivate2FA::class,
+        ExportAll::class,
+        GetVersion::class,
+        ImportCSV::class,
+        ImportVCards::class,
+        LangGenerate::class,
+        PingVersionServer::class,
+        SendReminders::class,
+        SendStayInTouch::class,
+        SentryRelease::class,
+        SetupProduction::class,
+        SetupTest::class,
+        SetupFrontEndTest::class,
+        SetPremiumAccount::class,
+        SetUserAdmin::class,
+        Update::class,
+        MigrateDatabaseCollation::class,
```

Comment:
We should sort this list alphabetically, could you just do it?

Reply:
Is done. But I hope that in the future, other developers will follow the sorting when adding commands)